names.txt

```diff
@@ -3,6 +3,13 @@ Abdulbasit Yusuf
 Allename Anthony
 Emmanuel Oloke
 Mamodu Joshua
+<<<<<<< HEAD
```

Comment:
Remove this line and other ones with non-name characters.

Reply:
done